test/parallel/test-http2-compat-serverrequest-pipe.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: can you please move this after the `common.hasCrypto` check?

Reply:
Uh..now after the update there are two `fixtures`. I don't think it can run with two `const fixtures` defined? Can you remove this line?